java/src/main/native/ai_onnxruntime_OnnxRuntime.c

```diff
@@ -54,3 +54,19 @@ JNIEXPORT jobjectArray JNICALL Java_ai_onnxruntime_OnnxRuntime_getAvailableProvi
   }
   return providerArray;
 }
+
+/*
+ * Class:     ai_onnxruntime_OnnxRuntime
+ * Method:    initialiseVersion
+ * Signature: ()Ljava/lang/String;
+ */
+JNIEXPORT jstring JNICALL Java_ai_onnxruntime_OnnxRuntime_initialiseVersion
+  (JNIEnv * jniEnv, jclass clazz) {
+  (void)clazz;  // required JNI parameter not needed by functions which don't access their host class.
+  const char* version = OrtGetApiBase()->GetVersionString();
+  jstring versionStr = NULL;
+  if (version != NULL) {
```

Comment:
> if (version != NULL) { [](http://example.com/codeflow?start=1&length=23)

This will neve be null. Redundant check. assert() might be more appropriate, meaning something is wrong with the build.

Reply:
Ok, I'll remove that and see if the asserts are available from there.